### ISSUE TITLE
Optional transformation, maybe use as babel macro

### ISSUE THREAD

**Issue by DimaRGB** [faceyspacey/babel-plugin-universal-import#93]
Question:
---
Is it possible to make this plugin work as [macro](https://github.com/kentcdodds/babel-plugin-macros), like [preval](https://github.com/kentcdodds/babel-plugin-preval#use-with-babel-plugin-macros) ?

Problem:
---
In some files i need to use default dynamic `import()`.
And i want to have some mechanism to enable the transform plugin only for some files, maybe even only for some cases.

**Comment by cdoublev** [comment]
Might be fixed by #134.

**Comment by ScriptedAlchemy** [comment]
Looking for maintainers on the universal suite. A significant amount of time is dedicated to module Federation and webpack 5 - this would near eliminate the need for Babel transformers this substantial.

**Comment by cdoublev** [comment]
I read your post(s) about it. It sounds very interesting. I didn't realize it could fill this space (code transformers). I'm not sure if I can take any new responsibilities with your universal packages. I'm already struggling to do enough work on my own to be honest + I'm not 100% fluent in english. But I can help when I can. This is the minimum I can do in return of accessing those great packages for free.

(Sorry to pollute this issue).

**Comment by ScriptedAlchemy** [comment]
Yeah no problem. And your PRs are appreciated! Until a point in time where there's a crossroads of needing to rebuild the universal toolset - it still does what's it needs to. 

The only part I might update would be universal component. But that would be a breaking change. Moving universal to a module federation based system since it's far superior and covers the basics of what universal did.
